.circleci/config.yml

```diff
@@ -355,6 +353,24 @@ jobs:
             docker push ${DOCKERHUB_REPO}:build_test
             docker push ${DOCKERHUB_REPO}:build_ui
             docker push ${DOCKERHUB_REPO}:latest
+  cirrus_deploy:
+    working_directory: ~/cirrus
+    machine:
+      image: ubuntu-2004:2023.04.2 # Ubuntu 20.04, Docker v20.10.24, Docker Compose v2.17.2
+      docker_layer_caching: true
+    steps:
+      - checkout
+      - deploy:
+          name: Deploy to latest
+          command: |
+            docker login -u $DOCKER_USER -p $DOCKER_PASS
+            make cirrus_build cirrus_test
+            docker tag cirrus:dev ${DOCKERHUB_CIRRUS_REPO}:cirrus_build
+            docker tag cirrus:test ${DOCKERHUB_CIRRUS_REPO}:cirrus_test
+            docker tag cirrus:deploy ${DOCKERHUB_REPO}:latest
+            docker push ${DOCKERHUB_CIRRUS_REPO}:cirrus_build
+            docker push ${DOCKERHUB_CIRRUS_REPO}:cirrus_test
+            docker push ${DOCKERHUB_CIRRUS_REPO}:latest
```

Comment:
Couple things

1) We don't need to push anything other than deploy, the only reason we push the other stages for experimenter is so in theory your local build can use the remote cache, which worked for a while then stopped and I haven't had time to go back and figure it out, so just build and push deploy here

2) We should be able to reuse all the existing environment variables except for repo, in theory we should be able to push to `mozilla/cirrus`, ie those creds should have permission to that org.  Give that a shot.